app/src/main/java/space/taran/arkbrowser/utils/FullscreenHelper.kt

```diff
@@ -0,0 +1,24 @@
+package space.taran.arkbrowser.utils
+
+import android.view.View
+import android.view.Window
+
+object FullscreenHelper {
+    fun setSystemUIVisibility(isVisible: Boolean, window: Window) {
+        if (isVisible) showSystemUI(window) else hideSystemUI(window)
+    }
+
+    fun hideSystemUI(window: Window) {
+        window.decorView.systemUiVisibility =
+            (View.SYSTEM_UI_FLAG_IMMERSIVE_STICKY
+ //                   or View.SYSTEM_UI_FLAG_LAYOUT_STABLE
```

Comment:
Yes, the code still contains a lot of commented-out code which is left from closed-source time.
But let's not produce it anymore — if there is something what might be useful in the future, it can be found using Git.